ants/registration/apply_transforms.py

```diff
@@ -132,6 +135,18 @@ def apply_transforms(
             if (moving.dimension == 4) and (fixed.dimension == 3) and (imagetype == 0):
                 raise Exception("Set imagetype 3 to transform time series images.")
 
+            # check that the user actually submitted a label image
+            # transforming intensity images with label interpolators can take a very long time
+            if interpolator in ["nearestNeighbor", "multiLabel", "genericLabel"]:
+                # if more than 5% of voxels are unique, moving probably isn't a label image
+                if len(np.unique(moving)) > np.prod(moving.shape) * 0.05:
```

Comment:
I think this needs to be `len(np.unique(moving.flatten()))`, because `np.unique(moving)` returns an antsImage and it's length is 1